elastio-nat-provision-lambda/cloudformation-nat.yaml

```
@@ -0,0 +1,44 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/awslabs/goformation/master/schema/cloudformation.schema.json
+AWSTemplateFormatVersion: '2010-09-09'
+Description: Deploys a single NAT Gateway
+
+Parameters:
+  SubnetId:
+    Type: String
+    Description: ID of a public subnet where the NAT Gateway will be deployed
+  RouteTableId:
+    Type: String
+    Description: ID of a route table associated with the subnet
```

Make it clear that there two different subnets involved here.
```suggestion
  PublicSubnetId:
    Type: String
    Description: ID of a public subnet where the NAT Gateway will be deployed
  PrivateSubnetRouteTableId:
    Type: String
    Description: ID of a route table associated with the private subnet that will be modified to forward internet traffic through NAT
```